### PR TITLE
Exclude unnecessary GPL dependency

### PR DESCRIPTION
**Pull request by ignasi35**

### CODE REVIEW

**Comment by raboof** [comment]
(to be exact: it looks like this dependency is "either GPL or CDDL", and the CDDL is generally acceptable, but we don't actually rely on this dependency so it's cleaner to remove it entirely)

File: project/Dependencies.scala [hunk 1]
```diff
@@ -51,7 +51,7 @@ object Dependencies {
 
     val plexusBuildApi = "org.sonatype.plexus" % "plexus-build-api" % "0.0.7" % "optional"// Apache v2
 
-    val play = "com.typesafe.play" %% "play" % Versions.play // Apache M2
+    val play = "com.typesafe.play" %% "play" % Versions.play exclude("javax.activation", "javax.activation-api")
```
Comment (ignasi35) [inline project/Dependencies.scala]:
```suggestion
    val play = "com.typesafe.play" %% "play" % Versions.play exclude("javax.activation", "javax.activation-api")  // Apache M2 (exclusion is "either GPL or CDDL")
```

Comment (ignasi35) [reply project/Dependencies.scala]: @raboof WDYT?

Comment (raboof) [reply project/Dependencies.scala]: :+1: .

is the `M2` intentional?

Comment (ignasi35) [reply project/Dependencies.scala]: 🤷🏼‍♂️

**Comment by raboof** [comment]
Excellent let's merge when green

**Pull request merged**